satpy/multiscene.py

```diff
@@ -341,17 +341,20 @@ def load_data(q):
 
         input_q = Queue(batch_size if batch_size is not None else 1)
         load_thread = Thread(target=load_data, args=(input_q,))
+        # set threads to daemon so they are killed if error is raised from main thread
+        load_thread.daemon = True
```

Comment:
This can be set in the `Thread` init right?

Reply:
Yes indeed. Changed it.